@here/harp-mapview/lib/poi/PoiRenderer.ts

```diff
@@ -152,15 +155,14 @@ class PoiBatch {
         const mesh = boxBuffer.mesh;
         mesh.frustumCulled = false;
 
-        const layer = this.textCanvas.addLayer(renderOrder);
-        layer.storage.scene.add(mesh);
+        const layer = this.m_poiRenderer.addLayer(renderOrder);
```

Comment:
It'd make more sense to create the layers in PoiRenderer, since it owns them. Then you only need to pass down the layer

Reply:
done